.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -907,6 +943,9 @@ stages:
 
     - script: tracer\build.cmd BuildAspNetIntegrationTests RunWindowsTracerIisIntegrationTests -Framework $(framework) --code-coverage
       displayName: RunWindowsIisTracerIntegrationTests
+      env:
+        DD_LOGGER_DD_API_KEY: $(ddApiKey)
+        DD_LOGGER_ENABLED: false
```

Comment:
Hmmm, shouldn't we enable the logger here? 🤔

Reply:
This is the one that require futher investigation, when we enable it the tests crashes.